### ISSUE TITLE
Add Design Systems project to VRMS

### ISSUE THREAD

**Issue by Hanastevenson** [hackforla/VRMS#763]
### Overview
We need to add/change the team meeting time for [Design Systems] so it will appear correctly on hackforla.org

### Is your team meeting team already displayed on hackforla.org?
- [ ] yes
- [x] no

### List all of your meeting times (PST time ONLY) (Name of project, name of team meeting, day of week, start time am/pm, duration):

- Design Systems Weekly meeting. Monday 6:30pm, 1 hour
- Design Systems UX/UI meeting.Tuesday 6:30pm 1 hour
- Design Systems UX Research meeting. Tuesday 11am 1 hour

Note: Please explicitly state if a meeting time that is currently displayed is to be deleted.

### What is your project page URL?
https://www.hackforla.org/projects/ - AWAITING.

### Who should we contact on your team (via Slack) for user acceptance testing
Slack Channel #name: design-systems 
Slack @handle: @hana stevenson
<hr />

**Comment by Hanastevenson** [comment]
@ExperimentsInHonesty This has been completed. Times added to this [ticket ](https://github.com/hackforla/design-systems/issues/105) Shall I close this Issue now?

- Design Systems Weekly meeting. Monday 6:30pm, 1 hour
- Design Systems UX/UI meeting.Tuesday 6:30pm 1 hour
- Design Systems UX Research meeting. Tuesday 11am 1 hour

**Comment by drubgrubby** [comment]
@Hanastevenson => I have added the Design Systems meeting times to the calendar.  Can you please check them and confirm if they are correct.  

https://www.hackforla.org/events/

Also, because this is a new project being added, there is some additional information that, if you can provide it, will fill out the profile in the database.  

The most important it the HfLA website URL, which I think you are still waiting on.  Once we have that, then the link on the calendar will go to your projects page.  In the meantime, if there's someplace you would like to have that linked to, let me know. 

Here's the other info I can add to the db, if you have it: 

partners:
name:"Design Systems"
description:
githubIdentifier:
githubUrl:
slackUrl:"https://hackforla.slack.com/archives/CH2U1CB9Q"
googleDriveUrl:
googleDriveId:
hflaWebsiteUrl:
videoConferenceLink:

No problem if you don't, but if you send me what you have I'll add it to the db.

**Comment by Hanastevenson** [comment]
As requested:

partners: No partner, it is a HfLA initiative.
name:"Design Systems"
description: The Design System initiative seeks to empower volunteers with the tools, documentation and templates for creating and maintaining a design system for their HfLA projects.  As Hack for LA continues to scale it has become more essential to create consistent documentation and standards for deliverables. 
githubIdentifier:
githubUrl: https://github.com/hackforla/design-systems
slackUrl:"https://hackforla.slack.com/archives/CH2U1CB9Q"
googleDriveUrl: https://drive.google.com/drive/u/0/folders/0ACLGQ8FFW2MCUk9PVA
googleDriveId: 0ACLGQ8FFW2MCUk9PVA
hflaWebsiteUrl: Awaiting (should be on the site within the next week)
videoConferenceLink: https://us02web.zoom.us/j/88447401964?pwd=cUkyNFNKVGZRckxzRkdrT1luVVJ3dz09

**Comment by drubgrubby** [comment]
@Hanastevenson -> Thanks. I've added that all to the db.  I'm going to close this issue, but either open a new issue with VRMS, or just slack me when you have the website url.
